### ISSUE TITLE
wired training result when switch from qwen2.5-7b-instruct to qwen3-8b

### ISSUE THREAD

**Issue by Raphrain** [volcengine/verl#1615]
## my training script:
python3 -m verl.trainer.main_ppo \
    algorithm.adv_estimator=grpo \
    data.train_files=/raphealhuang/simpleRL/train.parquet \
    data.val_files=/raphealhuang/simpleRL/test.parquet \
    data.train_batch_size=1024 \
    data.max_prompt_length=512 \
    data.max_response_length=1024 \
    data.filter_overlong_prompts=True \
    data.truncation='error' \
    actor_rollout_ref.model.path=/raphealhuang/models/Qwen3-8B \
    actor_rollout_ref.actor.optim.lr=1e-6 \
    actor_rollout_ref.model.use_remove_padding=True \
    actor_rollout_ref.actor.ppo_mini_batch_size=384 \
    actor_rollout_ref.actor.ppo_micro_batch_size_per_gpu=12 \
    actor_rollout_ref.actor.use_kl_loss=True \
    actor_rollout_ref.actor.kl_loss_coef=0.0001 \
    actor_rollout_ref.actor.kl_loss_type=low_var_kl \
    actor_rollout_ref.actor.entropy_coeff=0 \
    actor_rollout_ref.model.enable_gradient_checkpointing=True \
    actor_rollout_ref.actor.fsdp_config.param_offload=False \
    actor_rollout_ref.actor.fsdp_config.optimizer_offload=False \
    actor_rollout_ref.rollout.log_prob_micro_batch_size_per_gpu=80 \
    actor_rollout_ref.rollout.tensor_model_parallel_size=4 \
    actor_rollout_ref.rollout.name=vllm \
    actor_rollout_ref.rollout.gpu_memory_utilization=0.6 \
    actor_rollout_ref.rollout.n=6 \
    actor_rollout_ref.ref.log_prob_micro_batch_size_per_gpu=80 \
    actor_rollout_ref.ref.fsdp_config.param_offload=True \
    algorithm.use_kl_in_reward=False \
    custom_reward_function.path=/raphealhuang/verl/verl/utils/reward_score/simplelr_qwen.py \
    trainer.critic_warmup=0 \
    trainer.logger=['console','wandb'] \
    trainer.project_name='verl_train' \
    trainer.experiment_name='grpo_qwen3_8b' \
    trainer.n_gpus_per_node=8 \
    trainer.nnodes=4 \
    trainer.save_freq=5 \
    trainer.test_freq=5 \
    trainer.resume_mode='auto' \
    trainer.default_local_dir=/raphealhuang/models/checkpoints/${PROJECT_NAME}/${EXPERIMENT_NAME} \
    trainer.max_actor_ckpt_to_keep=2 \
    trainer.max_critic_ckpt_to_keep=2 \
    trainer.total_epochs=100 $@

## the output of the validation generations of qwen3 are of non-sense and the accuracy is always 0

[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m onnedesdesdesdesdesdesdesdesXTloggeddes ButdesdesXTdesdesXTdesdesXTdesXTloggeddesdesdesdesXTXTgingdesdesXT furn ApdesdesdesdesXTdesdesXTdesXTinatingdes showdes showdes showdes).
[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m
[36m(TaskRunner pid=17538)[0m AngledesdesdesdesXTinatingdes symiserdesdesXTinatingdesdes ButdesdesXTinatingdesdesdesdesXT furndesgingdesdesXTinatingdes sym thickdes symdes symdesdesnumericdes symdesdesinatingdesdes symdes symdesinatingdesdesnumericdesBrowserdes symdes symdes symdes symdesoltagedes symdes symdes symdes symdes symdes symdes Yetdes symdesdes symdes symdesdes symdes symdesdes symdesdes symdesdesdesnumericdes Babydesdes symdesdesnumericdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdes peersdesdesBrowserdes Babydesdes peersdesdes peersdesdes Babydesdes peersdesdes Babydesdes Babydesdes Babydesdesdes peersdesdes Babydesdesdes peersdesdesdes Babydesdesdesdes peersdesdesdesdesdesdesdesdesdesdesdesdesdesdesdesdesdesdesXT furndesо� expensedes symPickerEncodeEncode�unicipо� expense achieve GOPdes DbTodoTodostedesо� expenseinating institutionsdes symgingdes symо� expense sym sym sym sym BabyRelateddes sym sym sym Baby BusPicker[MAX myst schedules repository.printStackTrace DbTodo showdes symTodo institutions Db showdes Butdes Db showdes Butdes Butdes Butdes Butdesdes ButdesdesdesdesdesdesdesdesdesdesdesXT furndesо�ejdes sym sym.aldes[MAX Butdes ButdesdesdesdesdesdesdesXT ButdesdesdesdesdesdesXT ButdesdesdesXT
[36m(TaskRunner pid=17538)[0m [ground_truth] -2 + 7i
[36m(TaskRunner pid=17538)[0m [score] -1.0
[36m(TaskRunner pid=17538)[0m [acc] False
[36m(TaskRunner pid=17538)[0m ("Initial validation metrics: {'val-aux/simplelr_qwen/reward/mean@1': -1.0, "
[36m(TaskRunner pid=17538)[0m "'val-aux/simplelr_qwen/score/mean@1': -1.0, "
[36m(TaskRunner pid=17538)[0m "'val-core/simplelr_qwen/acc/mean@1': 0.0}")

<img width="475" alt="Image" src="https://github.com/user-attachments/assets/099f378f-a97e-489b-b0bb-a3bda0d385f5" />

## I have looked into the script provided as the example, but found nothing different to cause the training failure

**Comment by 0205090923** [comment]
same problem

**Comment by tjoymeed** [comment]
anybody solved the problem?

**Comment by maxx06** [comment]
just ran into this problem--try updating your vllm

minimum vllm version needs to be 0.8.4 for qwen3. i was running 0.8.2 and was running into this output. fixed itself after updating to 0.8.4. hope this helps!

**Comment by Kirrito-k423** [comment]
定时清理检查结论：该 issue 最近一次由当前运行账号（Kirrito-k423）之外账号产生的公开回复或公开引用是 2025-07-10T07:36:22Z，来源为 comment（maxx06）：https://github.com/verl-project/verl/issues/1615#issuecomment-3056083029。截至 2026-07-06，该时间已经超过 6 个月。

我已检查该 issue 的公开 comments 和 timeline（issue 页面：https://github.com/verl-project/verl/issues/1615；comments/timeline 对应 GitHub 公开 issue 记录）。根据本次定时清理规则，这类长时间没有新的非本人公开互动的 open issue 将先关闭。如果作者或社区成员仍能提供新的复现、证据或需求背景，请继续评论请求重开，或创建新的 issue 并引用这里的上下文。

---

Scheduled cleanup result: the latest public reply or public reference from an account other than the current runner (Kirrito-k423) was at 2025-07-10T07:36:22Z, from comment by maxx06: https://github.com/verl-project/verl/issues/1615#issuecomment-3056083029. As of 2026-07-06, that is more than 6 months ago.

I checked the public comments and timeline for this issue (issue page: https://github.com/verl-project/verl/issues/1615; comments/timeline are part of the public GitHub issue record). Under the scheduled cleanup rule, open issues with no recent non-runner public interaction are closed. If the author or community can still provide a fresh reproduction, evidence, or requirement context, please comment to request reopening or open a new issue and link this context.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
